### PR TITLE
feat(config): add more theme configuration for UI elements

### LINKED ISSUE #389: Feature Request: Abiity to change the text color of chat messages / channel (and friend) titles
Hi all,

I may be blind, but I don't see a way to edit the text color of chat messages and server/DM titles in the config file? I see you can change the color of the server/DM heading (Guilds), and the author color and border color, but I don't see an option for chat text, server/DM text list color, or for the highlight color of a selected channel?

On my laptop, the channel I have selected gets highlighted, but is highlighted in white and the text is white, so it completely blends in and you can't read what channel you are viewing (see attached screenshot).

Should be a fairly easy feature to add to allow customizable colors for those? :)

Cheers and again, thanks for the hard work....great little program! :)
Screenshot_2024-04-22_14-12-18

### PR DESCRIPTION
**Pull request by b2ag**
Hi,

I added some more config options for text colors. It may fix #389 and definitely allows me to read the text (and not have white text on white background as before). 

Best,

### CODE REVIEW

File: internal/config/theme.go [hunk 1]
```diff
@@ -16,12 +16,16 @@ type (
 	}
 
 	GuildsTreeTheme struct {
-		AutoExpandFolders bool `toml:"auto_expand_folders"`
-		Graphics          bool `toml:"graphics"`
+		AutoExpandFolders bool   `toml:"auto_expand_folders"`
+		Graphics          bool   `toml:"graphics"`
+		DmColor           string `toml:"dm_color"`
```
Comment (ayn2op) [inline internal/config/theme.go]: Rename `dm_color` to `private_channel_color`.

File: internal/config/theme.go [hunk 2]
```diff
@@ -16,12 +16,16 @@ type (
...
+		FolderColor       string `toml:"folder_color"`
```
Comment (ayn2op) [inline internal/config/theme.go]: Rename `folder_color` to `guild_color`.

File: internal/config/theme.go [hunk 3]
```diff
@@ -16,12 +16,16 @@ type (
...
+		ChannelColor      string `toml:"channel_color"`
 	}
 
 	MessagesTextTheme struct {
 		AuthorColor    string `toml:"author_color"`
+		TextColor      string `toml:"text_color"`
```
Comment (ayn2op) [inline internal/config/theme.go]: Rename `text_color` to `content_color`.

File: internal/config/theme.go [hunk 4]
```diff
@@ -16,16 +16,16 @@ type (
...
 
 	GuildsTreeTheme struct {
-		AutoExpandFolders bool   `toml:"auto_expand_folders"`
-		Graphics          bool   `toml:"graphics"`
-		DmColor           string `toml:"dm_color"`
-		FolderColor       string `toml:"folder_color"`
-		ChannelColor      string `toml:"channel_color"`
+		AutoExpandFolders   bool   `toml:"auto_expand_folders"`
+		ChannelColor        string `toml:"channel_color"`
+		Graphics            bool   `toml:"graphics"`
```
Comment (ayn2op) [inline internal/config/theme.go:21]: Why is the `Graphics` field shifted down?

Comment (b2ag) [reply internal/config/theme.go:21]: sorted alphabetically

Comment (b2ag) [reply internal/config/theme.go:21]: Should I change it back?

**Comment by Retropaint** [comment]
Don't forget to add defaults to the readme config!

**Review by ayn2op — APPROVED** [review]
LGTM! <3

**Comment by dagleykd** [comment]
Thanks peeps! This helps! :D

**Pull request merged**